### ISSUE TITLE
Incremental compilation has been disabled: it is not compatible with whole module optimization Command CompileSwiftSources failed with a nonzero exit code

### ISSUE THREAD

**Issue by Pherne** [dlutton/flutter_tts#317]
## 🐛 Bug Report

Since upgrading to XCode 13.3 I get this this error in this package and flutter_inappwebview when archiving my project. It happens on my mac and when I build with Codemagic. I don't get this error on 13.2 on my mac and when building with Codemagic. I checked my configuration according to this post https://developer.apple.com/forums/thread/701981 and nothing  seems to work. However I don't think I can change any XCode settings aside from the version in Codemagic anyway, so I don't think it has anything to do with my configuration.

### Expected behavior
archving with XCode 13.3 should be successful

### Reproduction steps
trying to archive a project that uses this package

### Configuration

**Version:** 3.3.3

**Flutter version:** stable 2.10.3

**XCode version:** 13.3

**Platform:** 
- [x] :iphone: iOS
- [ ] :robot: Android

**Comment by dlutton** [comment]
@Pherne does your app work successfully with other flutter packages?

**Comment by Pherne** [comment]
Yes, I have other packages installed like provider, flutter_html, several firebase packages. None of them show this error.

**Comment by Pherne** [comment]
Just in case: I have another flutter project that doesn't use flutter_tts or flutter_inappwebview, but does include provider and multiple firebase packages and then some and I can archive that on 13.3 without issue.

**Comment by dlutton** [comment]
@Pherne does `pod update` work for you?
